.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+# Note: The directories key (plural) supports globbing and wildcards, unlike directory (singular).
+# See: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#directories-or-directory--
+
+updates:
+  - package-ecosystem: "bun"
```

Comment:
@salmanfarisvp, see this for the list of available values for the `package-ecosystem` field.

Reply:
Hi, Can you share more details and usage please.

Reply:
I forgot to add the link: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#directories-or-directory--